src/views/SignupPage.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import toastr from 'toastr';
+import SignupForm from '../components/SignupForm';
+import userSignupAction from '../actions/signupActions';
+
+class SignupPage extends Component {
+  constructor(props) {
+    super(props);
+    this.InitialState = {
+      username: '',
+      firstName: '',
+      lastName: '',
+      password: '',
+      confirmPassword: '',
+      email: '',
+    };
+    this.state = { ...this.InitialState, isRequestSent: false };
+    this.isRequestSent = false;
+    this.handleOnChange = this.handleOnChange.bind(this);
```

Comment:
Remove bind statements.
Use arrow function to create class methods
Eg. `handleOnChange = (event) => {...}` with this, there's no need for a `bind` statement in the constructor.

Reply:
I have implemented this feedback